demos/link-prediction/random-walks/cora-lp-demo.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
<p>Maybe the functions here could be marked to correspond to the lists steps to give the user a little more guidance for this large cell? E.g.</p><p> </p><pre class="ql-syntax" spellcheck="false"># 1. link embeddings
...

# 2. training classifier
...

# 3. evaluate classifier
...
</pre><p>Or maybe that doesn't make sense?</p><p> </p><p>Also in <code>link_examples_to_features</code>, I think the <code>isinstance(binary_operator, str)</code> support code isn't used and the <code>operator_func</code> dict isn't defined.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/1190/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<blockquote>Maybe the functions here could be marked to correspond to the lists steps to give the user a little more guidance for this large cell?</blockquote><p>Yeah that makes sense to me</p><p> </p><blockquote>Also in&nbsp;<code>link_examples_to_features</code>, I think the&nbsp;<code>isinstance(binary_operator, str)</code>&nbsp;support code isn't used and the&nbsp;<code>operator_func</code>&nbsp;dict isn't defined.</blockquote><p>good catch!</p>